model/labels/labels_stringlabels.go

```diff
@@ -701,3 +701,8 @@ func NewScratchBuilderWithSymbolTable(_ *SymbolTable, n int) ScratchBuilder {
 func (b *ScratchBuilder) SetSymbolTable(_ *SymbolTable) {
 	// no-op
 }
+
+// SizeOfLabels returns the approximate space required for n copies of a label.
+func SizeOfLabels(name, value string, n uint64) uint64 {
```

Comment:
maybe we can have a common util to avoid doing the same as in `func labelSize()`

Reply:
I changed it to call `labelSize`.